packages/route-plugin/src/react.tsx

```diff
@@ -1,62 +1,8 @@
 import { connect } from "@prodo/core";
-import * as pathToRegexp from "path-to-regexp";
 import * as React from "react";
 import * as actions from "./actions";
 import { RouteParams } from "./types";
-import { createParamString } from "./utils";
-
-const cache = {};
-const cacheLimit = 1000;
-let cacheCount = 0;
-
-const compilePattern = (
-  pattern: string,
-  options: pathToRegexp.RegExpOptions,
-) => {
-  const optionKey = `${options.end}.${options.strict}.${options.sensitive}`;
-  const optionCache = cache[optionKey] || (cache[optionKey] = {});
-
-  if (optionCache[pattern]) {
-    return optionCache[pattern];
-  }
-
-  const keys = [];
-  const regex = pathToRegexp(pattern, keys, options);
-  const result = { keys, regex };
-
-  if (cacheCount < cacheLimit) {
-    optionCache[pattern] = result;
-    cacheCount++;
-  }
-
-  return result;
-};
-
-const routeMatches = (
-  path: string,
-  pattern: string,
-  exact: boolean = false,
-) => {
-  const options = { end: exact, strict: false, sensitive: false };
-  const { regex, keys } = compilePattern(pattern, options);
-
-  const match = regex.exec(path);
-  if (match == null) {
-    return null;
-  }
-
-  const [url, ...values] = match;
-  const isExact = url === path;
-
-  if (exact && !isExact) {
-    return null;
-  }
-
-  return keys.reduce(
-    (acc, key, index) => ({ ...acc, [key.name]: values[index] }),
-    {},
-  );
-};
+import { createParamString, matchRoute } from "./utils";
```

Comment:
Moving and renaming functions like matchRoute would make it harder for someone to diff prodo-kanban with the original kanban,

Reply:
I don't follow. This function is in the `@prodo/route` plugin - it doesn't exist in react-kanban and is simply used in the kanban example.

Reply:
oh, sorry, ignore me - I mistakenly thought you where moving code around in example/kanban